ganga/GangaDirac/Lib/RTHandlers/RunTimeHandlerUtils.py

```diff
@@ -86,9 +86,10 @@ def master_sandbox_prepare(app, appmasterconfig, sharedir_roots=None):
             from GangaCore.GPIDev.Lib.Job import JobError
             raise JobError("InputFiles have been requested but there are objects in the inputSandBox... Aborting Job Prepare!")
         inputsandbox = []
-        for filepattern in getInputFilesPatterns(job)[0]:
+        (fileNames, tmpDir) = getInputFilesPatterns(job)
```

Comment:
This looks like a change in the output of the `getInputFilesPatterns` i.e. from a list to a tuple(list, str). Is this change already committed as it's not part of this PR

Reply:
It is not a change to `getInputFilesPatterns`, more that we want to know the location of the tmpDir to remove it. The parentheses were just because I copied another bit of code that uses this function.

Reply:
sorry, my bad I missed the `[0]` on the end of the line.